.buildkite/scripts/pipelines/security_solution_quality_gate/pipeline.sh

```diff
@@ -13,7 +13,6 @@ echo "--- Serverless Security Second Quality Gate"
 cd x-pack/test/security_solution_cypress
 set +e
 
-VAULT_DEC_KEY=$(vault read -field=key secret/kibana-issues/dev/security-solution-qg-enc-key)
-ENV_PWD=$(echo $TEST_ENV_PWD | openssl aes-256-cbc -d -a -pass pass:$VAULT_DEC_KEY)
+QA_API_KEY=$(vault read -field=api-key secret/kibana-issues/dev/security-solution-qg-enc-key)
```

Comment:
We occasionally have timeouts with vault, so we typically wrap vault reads in a retry loop.

```suggestion
QA_API_KEY="$(retry 5 5 vault read -field=api-key secret/kibana-issues/dev/security-solution-qg-enc-key)"
```

Reply:
Fixed with [36eba9828c9b6839d7aadc3a445b5ac0c20b9e16](https://github.com/elastic/kibana/commit/36eba9828c9b6839d7aadc3a445b5ac0c20b9e16)

Thanks for letting me know!!